src/EventSourcingTests/Examples/TeleHealth/BoardViewProjection.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Marten;
+using Marten.Events;
+using Marten.Events.Aggregation;
+using Marten.Events.Projections;
+
+namespace EventSourcingTests.Examples.TeleHealth;
+
+#if NET6_0_OR_GREATER
+public class BoardViewProjection: CrossStreamAggregateProjection<BoardView, Guid>
+{
+    protected override ValueTask GroupEvents(IEventGrouping<Guid> grouping, IQuerySession session, List<IEvent> events)
+    {
+        grouping.AddEventsWithMetadata<BoardStateEvent>(e => e.StreamId, events);
+        grouping.AddEvents<IBoardEvent>(x => x.BoardId, events);
```

Comment:
**QUESTION:** Is this intentional that we don't have any implementation of `IBoardEvent`?